packages/constants/src/lib/constants/endpoints.ts

```diff
@@ -0,0 +1,52 @@
+export enum LIT_ENDPOINT_VERSION {
+  LEGACY = '/',
```

Comment:
`LEGACY` might not be the best wording here as only `pkpSign` and `executeJS` have new `v1` routes to my understanding from reviews on the node implementations for our new authentication unification efforts. Does it make sense to you to change `LEGACY` to `V0` or is that more confusing?